### DISCUSSION TITLE
"unknown shell type" error when "source diracos env" in bash scripts

### DISCUSSION THREAD

**Discussion by zhangxiaomei** [DIRACGrid/DIRAC#7373] [Q&A]
In an executable script (test.sh) with "source DIRAC OS environment":   
#!/bin/bash
source /cvmfs/dcomputing.ihep.ac.cn/dirac/IHEPDIRAC/v3r1/diracos/diracosrc

I got the error like the following:
(base) -bash-4.2$ ./test.sh
error    libmamba Please provide a shell type.
    Run with --help for more information.

critical libmamba Unknown shell type. Aborting.

How to avoid this error?

**Comment by andresailer** [comment]
This has already been reported https://github.com/DIRACGrid/DIRAC/issues/7260
